### PR TITLE
styled team page according to wireframe

### PR DESCRIPTION
**Pull request by aileen-ji**
API calls ran out, so there's hardcoded data to aid testing and visualization for now

### CODE REVIEW

File: src/contexts/home.jsx [hunk 1]
```diff
@@ -230,6 +230,7 @@ export const HomeContextProvider = ({ children }) => {
                     "x-rapidapi-key": SPORTS_API_KEY
                 }
             })
+            console.log(json)
```
Comment (LifeBringer) [inline src/contexts/home.jsx]: comment out

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Comment console.log out

File: src/contexts/home.jsx (hunk 1 above)
Comment (aileen-ji) [reply src/contexts/home.jsx]: Just commented it out!

**Review by LifeBringer — APPROVED** [review]
LGTM

**Pull request merged**